timbot.py

```diff
@@ -52,7 +54,9 @@ def choose_lunchcation():
 def run_timbot():
 	history = sc.api_call("groups.history", channel=channel_name, count=1)
 
-	handle_ideal_lunch_time(datetime.datetime.now().strftime('%H:%M'))
+	# handle ideal lunch time if it is a weekday
+	if datetime.datetime.today().weekday() != saturday_index_elem or datetime.datetime.today().weekday() != sunday_index_elem:
```

Comment:
I believe you can do something like this:
```suggestion
	if datetime.datetime.today().weekday() not in [saturday_index_elem, sunday_index_elem]:
```